src/Api/BoardApi.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+export default function Activity() {
+  const [WhatTodoList, setWhatTodoList] = useState([]);
```

Comment:
The component that calls this function should handle the state management. This should just be a pure function